Resources/Prototypes/Entities/Markers/Spawners/Food_Drinks/drinks_bottles.yml

```diff
@@ -0,0 +1,39 @@
+- type: entity
+  id: RandomDrinkBottle
+  name: random drink spawner
+  suffix: Bottle
+  parent: MarkerBase
+  components:
+  - type: Sprite
+    layers:
+      - state: red
+      - texture: Objects/Consumable/Drinks/ginbottle.rsi/icon.png 
+  - type: RandomSpawner
+  #small item
+    prototypes:
+      - DrinkAbsintheBottleFull
+      - DrinkBlueCuracaoBottleFull
+      - DrinkCognacBottleFull
+      - DrinkGrenadineBottleFull
+      - DrinkGinBottleFull
+      - DrinkGoldschlagerBottleFull
+      - DrinkKahluaBottleFull
+      - DrinkMelonLiquorBottleFull
+      - DrinkPatronBottleFull
+      - DrinkRumBottleFull
+      - DrinkTequilaBottleFull
+      - DrinkVermouthBottleFull
+      - DrinkVodkaBottleFull
+      - DrinkWhiskeyBottleFull
+      - DrinkWineBottleFull
+      - DrinkBeer
+      - DrinkAleBottleFull
+    chance: 0.8
+    offset: 0.0
+    #rare
+    rarePrototypes:
+      - DrinkPoisonWinebottleFull
+    rareChance: 0.01
+
+  placement:
+    mode: AlignTileAny
```

Comment:
this should go before the components list and without whitespace preceding it, same goes for all of the other instances of this in the other spawners yaml

Reply:
For the record, placement is placed at the end of some of the other spawners, which is why I did it that way but I will correct it, nbd.